### ISSUE TITLE
Constant DNS (DoT) to 1.1.1.1 & 1.0.0.1 (again...)

### ISSUE THREAD

**Issue by tescophil** [home-assistant/supervisor#2139]
**Home Assistant release with the issue:**

arch | armv7l
-- | --
chassis | embedded
dev | false
docker | true
docker_version | 19.03.11
hassio | true
host_os | HassOS 4.13
installation_type | Home Assistant OS
os_name | Linux
os_version | 4.19.127-v7
python_version | 3.8.5
supervisor | 247
timezone | Europe/London
version | 0.116.4
virtualenv | false

**Operating environment (HassOS/Generic):**

HassOS on Raspberry Pi 3B+

**Supervisor logs:**

```
20-10-15 07:20:21 INFO (MainThread) [supervisor.store] Load add-ons from store: 68 all - 0 new - 0 remove
20-10-15 07:27:10 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
20-10-15 07:57:12 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
20-10-15 08:06:18 INFO (MainThread) [supervisor.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup
20-10-15 08:07:53 INFO (MainThread) [supervisor.host.info] Update local host information
20-10-15 08:07:55 INFO (MainThread) [supervisor.host.services] Update service information
20-10-15 08:07:55 INFO (MainThread) [supervisor.host.network] Update local network information
20-10-15 08:07:57 INFO (MainThread) [supervisor.host.sound] Update PulseAudio information
20-10-15 08:27:13 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
20-10-15 08:57:15 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
20-10-15 09:06:19 INFO (MainThread) [supervisor.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup
20-10-15 09:19:01 INFO (MainThread) [supervisor.updater] Fetch update data from https://version.home-assistant.io/stable.json
20-10-15 09:27:16 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
20-10-15 09:57:18 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
20-10-15 10:06:20 INFO (MainThread) [supervisor.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup
20-10-15 10:14:37 INFO (MainThread) [supervisor.host.info] Update local host information
20-10-15 10:14:38 INFO (MainThread) [supervisor.host.services] Update service information
20-10-15 10:14:39 INFO (MainThread) [supervisor.host.network] Update local network information
20-10-15 10:14:40 INFO (MainThread) [supervisor.host.sound] Update PulseAudio information
20-10-15 10:20:21 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/sabeechen/hassio-google-drive-backup repository
20-10-15 10:20:21 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/hassio-addons/repository repository
20-10-15 10:20:21 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/home-assistant/hassio-addons repository
20-10-15 10:20:21 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/jeroenterheerdt/ring-hassio repository
20-10-15 10:20:23 INFO (MainThread) [supervisor.store] Load add-ons from store: 68 all - 0 new - 0 remove
20-10-15 10:27:20 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
20-10-15 10:57:21 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
20-10-15 11:06:22 INFO (MainThread) [supervisor.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup
20-10-15 11:19:01 INFO (MainThread) [supervisor.updater] Fetch update data from https://version.home-assistant.io/stable.json
20-10-15 11:27:23 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
20-10-15 11:57:25 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
20-10-15 12:06:23 INFO (MainThread) [supervisor.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup
20-10-15 12:21:20 INFO (MainThread) [supervisor.host.info] Update local host information
20-10-15 12:21:22 INFO (MainThread) [supervisor.host.services] Update service information
20-10-15 12:21:22 INFO (MainThread) [supervisor.host.network] Update local network information
20-10-15 12:21:23 INFO (MainThread) [supervisor.host.sound] Update PulseAudio information
20-10-15 12:27:26 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
```

Constant DoT DNS requests to 1.1.1.1, currently recording 15 requests per min which now makes my Home Assistant instance the device that generates the most sessions on my network (and I have over 50 devices online...). I'm seeing around 15 DoT requests/min from the supervisior. Raised this as an issue before (#1911). This functionality is a 'fall back' in case the user messes up their DNS. If this is the case, then this functionality is badly broken, moreover its a breach of privacy and should be removed all together.

Here are the DNS logs

```
~ $ ha dns logs
[INFO] 172.30.32.1:55017 - 42471 "PTR IN 145.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 127 0.003478746s
[INFO] 172.30.32.1:56332 - 36451 "PTR IN 244.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 125 0.004543328s
[INFO] 172.30.32.1:53905 - 18010 "PTR IN 204.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 111 0.003779631s
[INFO] 172.30.32.1:60580 - 49769 "PTR IN 35.10.168.192.in-addr.arpa. udp 44 false 512" NOERROR qr,aa,rd,ra 107 0.003462235s
[INFO] 172.30.32.1:42353 - 32877 "PTR IN 176.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 121 0.002831663s
[INFO] 172.30.32.1:53064 - 3792 "PTR IN 2.33.30.172.in-addr.arpa. udp 42 false 512" NOERROR qr,aa,rd 166 0.000900884s
[INFO] 172.30.32.1:45869 - 15440 "PTR IN 25.10.168.192.in-addr.arpa. udp 44 false 512" NOERROR qr,aa,rd,ra 112 0.002734215s
[INFO] 172.30.32.1:36709 - 5427 "PTR IN 182.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 126 0.00277682s
[INFO] 172.30.32.1:50609 - 50762 "PTR IN 224.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 122 0.002436351s
[INFO] 172.30.32.1:55056 - 64860 "PTR IN 113.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 119 0.002638174s
[INFO] 172.30.32.1:46324 - 19204 "PTR IN 230.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 118 0.002923747s
[INFO] 172.30.32.1:49793 - 47256 "PTR IN 109.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 123 0.002833278s
[INFO] 172.30.32.1:59078 - 63707 "PTR IN 110.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 129 0.004849838s
[INFO] 172.30.32.1:34177 - 31014 "PTR IN 157.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 122 0.002592028s
[INFO] 172.30.32.1:48518 - 13223 "PTR IN 1.10.168.192.in-addr.arpa. udp 43 false 512" NOERROR qr,aa,rd,ra 109 0.003639579s
[INFO] 172.30.32.1:45096 - 54726 "PTR IN 158.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 126 0.004779056s
[INFO] 172.30.32.1:41810 - 38137 "PTR IN 246.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 119 0.002828694s
[INFO] 172.30.32.1:39058 - 31885 "PTR IN 177.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 127 0.002947965s
[INFO] 172.30.32.1:38166 - 48724 "PTR IN 120.10.168.192.in-addr.arpa. udp 45 false 512" NXDOMAIN qr,rd,ra 45 0.015130972s
[INFO] 172.30.32.1:42721 - 31958 "PTR IN 178.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 130 0.002926507s
[INFO] 172.30.32.1:52241 - 1699 "PTR IN 3.33.30.172.in-addr.arpa. udp 42 false 512" NOERROR qr,aa,rd 202 0.002145102s
[INFO] 172.30.32.1:34076 - 20860 "PTR IN 195.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 129 0.005015775s
[INFO] 172.30.32.1:47587 - 59446 "PTR IN 125.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 123 0.003963225s
[INFO] 172.30.32.1:48107 - 45239 "PTR IN 0.33.30.172.in-addr.arpa. udp 42 false 512" NOERROR qr,aa,rd 148 0.000936197s
[INFO] 172.30.32.1:40907 - 19068 "PTR IN 126.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 122 0.003083746s
[INFO] 172.30.32.1:55271 - 22093 "A IN api.openweathermap.org. udp 40 false 512" NOERROR qr,rd,ra 192 0.00407864s
[INFO] 172.30.32.1:55271 - 24229 "AAAA IN api.openweathermap.org. udp 40 false 512" NOERROR qr,rd,ra 145 0.012053422s
[INFO] 172.30.33.1:39547 - 13031 "A IN www.duckdns.org. udp 33 false 512" NOERROR qr,rd,ra 247 0.012022745s
[INFO] 172.30.33.1:39531 - 14073 "AAAA IN www.duckdns.org. udp 33 false 512" NOERROR qr,rd,ra 224 0.013469566s
[INFO] 172.30.32.1:39494 - 28551 "AAAA IN mobile-apps.home-assistant.io. udp 47 false 512" NOERROR qr,rd,ra 47 0.004155932s
[INFO] 172.30.32.1:39494 - 26311 "A IN mobile-apps.home-assistant.io. udp 47 false 512" NOERROR qr,rd,ra 137 0.016381436s
[INFO] 172.30.32.1:39649 - 45015 "A IN share-service-download-bucket.s3.amazonaws.com. udp 64 false 512" NOERROR qr,rd,ra 180 0.014561647s
[INFO] 172.30.32.1:39649 - 47723 "AAAA IN share-service-download-bucket.s3.amazonaws.com. udp 64 false 512" NOERROR qr,rd,ra 248 0.100646841s
[INFO] 172.30.33.1:54355 - 35997 "AAAA IN www.duckdns.org. udp 33 false 512" NOERROR qr,rd,ra 224 0.014208053s
[INFO] 172.30.33.1:40425 - 34226 "A IN www.duckdns.org. udp 33 false 512" NOERROR qr,rd,ra 247 0.014164303s
[INFO] 172.30.32.1:59664 - 34116 "PTR IN 133.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 123 0.003438537s
[INFO] 172.30.32.1:55215 - 12231 "PTR IN 211.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 127 0.00309312s
[INFO] 172.30.32.1:51251 - 40070 "PTR IN 241.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 118 0.002699787s
[INFO] 172.30.32.1:47974 - 21653 "PTR IN 101.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 118 0.00296286s
[INFO] 172.30.32.1:47065 - 11156 "PTR IN 242.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 132 0.002720777s
[INFO] 172.30.32.1:37816 - 15694 "PTR IN 102.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 130 0.002881558s
[INFO] 172.30.32.1:56824 - 57854 "PTR IN 3.32.30.172.in-addr.arpa. udp 42 false 512" NOERROR qr,aa,rd 138 0.000847342s
[INFO] 172.30.32.1:55876 - 9868 "PTR IN 179.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 119 0.003775984s
[INFO] 172.30.32.1:42728 - 730 "PTR IN 1.33.30.172.in-addr.arpa. udp 42 false 512" NOERROR qr,aa,rd 156 0.000904322s
[INFO] 172.30.32.1:39575 - 11808 "PTR IN 197.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 119 0.003072495s
[INFO] 172.30.32.1:48834 - 2713 "PTR IN 198.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 126 0.005537961s
[INFO] 172.30.32.1:54903 - 6424 "PTR IN 2.0.17.172.in-addr.arpa. udp 41 false 512" NXDOMAIN qr,rd,ra 41 0.012922533s
[INFO] 172.30.32.1:38453 - 5612 "PTR IN 20.10.168.192.in-addr.arpa. udp 44 false 512" NOERROR qr,aa,rd,ra 106 0.002520569s
[INFO] 172.30.32.1:52604 - 39412 "PTR IN 165.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 125 0.002372028s
[INFO] 172.30.32.1:38869 - 57386 "PTR IN 223.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 120 0.002954892s
[INFO] 172.30.32.1:34665 - 45153 "PTR IN 253.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 120 0.002470413s
[INFO] 172.30.32.1:40337 - 38123 "PTR IN 145.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 127 0.0029001s
[INFO] 172.30.32.1:53882 - 37045 "PTR IN 244.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 125 0.002877079s
[INFO] 172.30.32.1:35884 - 43410 "PTR IN 204.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 111 0.002724528s
[INFO] 172.30.32.1:42205 - 28113 "PTR IN 35.10.168.192.in-addr.arpa. udp 44 false 512" NOERROR qr,aa,rd,ra 107 0.002929266s
[INFO] 172.30.32.1:35672 - 9847 "PTR IN 176.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 121 0.003159266s
[INFO] 172.30.32.1:54639 - 10904 "PTR IN 2.33.30.172.in-addr.arpa. udp 42 false 512" NOERROR qr,aa,rd 166 0.001602393s
[INFO] 172.30.32.1:33554 - 12005 "PTR IN 25.10.168.192.in-addr.arpa. udp 44 false 512" NOERROR qr,aa,rd,ra 112 0.003006089s
[INFO] 172.30.32.1:57071 - 49053 "PTR IN 182.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 126 0.003026298s
[INFO] 172.30.32.1:55417 - 61990 "PTR IN 224.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 122 0.002367809s
[INFO] 172.30.32.1:33843 - 4111 "PTR IN 113.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 119 0.003010621s
[INFO] 172.30.32.1:59224 - 7689 "PTR IN 230.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 118 0.003083433s
[INFO] 172.30.32.1:43916 - 5881 "PTR IN 109.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 123 0.00310609s
[INFO] 172.30.32.1:48377 - 34291 "PTR IN 110.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 129 0.002682704s
[INFO] 172.30.32.1:55310 - 60910 "PTR IN 157.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 122 0.003236298s
[INFO] 172.30.32.1:51109 - 11844 "PTR IN 1.10.168.192.in-addr.arpa. udp 43 false 512" NOERROR qr,aa,rd,ra 109 0.004396452s
[INFO] 172.30.32.1:52094 - 12408 "PTR IN 158.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 126 0.004594316s
[INFO] 172.30.32.1:59744 - 63488 "PTR IN 246.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 119 0.0032401s
[INFO] 172.30.32.1:39797 - 7698 "PTR IN 177.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 127 0.004208952s
[INFO] 172.30.32.1:33823 - 5085 "PTR IN 120.10.168.192.in-addr.arpa. udp 45 false 512" NXDOMAIN qr,rd,ra 45 0.015059926s
[INFO] 172.30.32.1:38412 - 3742 "PTR IN 178.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 130 0.003246297s
[INFO] 172.30.32.1:32853 - 33795 "PTR IN 3.33.30.172.in-addr.arpa. udp 42 false 512" NOERROR qr,aa,rd 202 0.001035832s
[INFO] 172.30.32.1:34489 - 25269 "PTR IN 195.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 129 0.002638434s
[INFO] 172.30.32.1:59840 - 8187 "PTR IN 125.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 123 0.002820048s
[INFO] 172.30.32.1:40553 - 58913 "PTR IN 0.33.30.172.in-addr.arpa. udp 42 false 512" NOERROR qr,aa,rd 148 0.000986249s
[INFO] 172.30.32.1:48725 - 52216 "PTR IN 126.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 122 0.002780205s
[INFO] 172.30.32.1:38642 - 14109 "AAAA IN mobile-apps.home-assistant.io. udp 47 false 512" NOERROR qr,rd,ra 47 0.003466922s
[INFO] 172.30.32.1:38642 - 12702 "A IN mobile-apps.home-assistant.io. udp 47 false 512" NOERROR qr,rd,ra 137 0.012744357s
[INFO] 172.30.32.1:55121 - 46053 "A IN api.openweathermap.org. udp 40 false 512" NOERROR qr,rd,ra 192 0.012315659s
[INFO] 172.30.32.1:55121 - 47980 "AAAA IN api.openweathermap.org. udp 40 false 512" NOERROR qr,rd,ra 145 0.01184592s
[INFO] 172.30.33.1:43592 - 12248 "A IN www.duckdns.org. udp 33 false 512" NOERROR qr,rd,ra 247 0.013775604s
[INFO] 172.30.33.1:50274 - 13290 "AAAA IN www.duckdns.org. udp 33 false 512" NOERROR qr,rd,ra 224 0.015652528s
[INFO] 172.30.32.1:60951 - 44751 "AAAA IN mobile-apps.home-assistant.io. udp 47 false 512" NOERROR qr,rd,ra 47 0.003018017s
[INFO] 172.30.32.1:60951 - 42928 "A IN mobile-apps.home-assistant.io. udp 47 false 512" NOERROR qr,rd,ra 137 0.004509265s
[INFO] 172.30.32.1:37578 - 3146 "AAAA IN share-service-download-bucket.s3.amazonaws.com. udp 64 false 512" NOERROR qr,rd,ra 248 0.014244093s
[INFO] 172.30.32.1:37578 - 385 "A IN share-service-download-bucket.s3.amazonaws.com. udp 64 false 512" NOERROR qr,rd,ra 180 0.10415776s
[INFO] 172.30.32.1:54362 - 37234 "AAAA IN mobile-apps.home-assistant.io. udp 47 false 512" NOERROR qr,rd,ra 47 0.003163641s
[INFO] 172.30.32.1:54362 - 35359 "A IN mobile-apps.home-assistant.io. udp 47 false 512" NOERROR qr,rd,ra 137 0.01350899s
[INFO] 172.30.32.1:47930 - 21952 "AAAA IN api.ring.com. udp 30 false 512" NOERROR qr,rd,ra 224 0.014731228s
[INFO] 172.30.32.1:47930 - 20181 "A IN api.ring.com. udp 30 false 512" NOERROR qr,rd,ra 688 0.01511628s
[INFO] 172.30.32.1:36087 - 4967 "A IN share-service-download-bucket.s3.amazonaws.com. udp 64 false 512" NOERROR qr,rd,ra 180 0.013208001s
[INFO] 172.30.32.1:36087 - 7624 "AAAA IN share-service-download-bucket.s3.amazonaws.com. udp 64 false 512" NOERROR qr,rd,ra 248 0.012855762s
[INFO] 172.30.33.1:35721 - 11822 "AAAA IN www.duckdns.org. udp 33 false 512" NOERROR qr,rd,ra 224 0.01337748s
[INFO] 172.30.33.1:39813 - 9947 "A IN www.duckdns.org. udp 33 false 512" NOERROR qr,rd,ra 247 0.014652321s
[INFO] 172.30.32.1:53962 - 26280 "PTR IN 133.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 123 0.003619839s
[INFO] 172.30.32.1:45694 - 1175 "PTR IN 211.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 127 0.003062027s
[INFO] 172.30.32.1:48487 - 55759 "PTR IN 241.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 118 0.003433849s
[INFO] 172.30.32.1:42542 - 64064 "PTR IN 101.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 118 0.005109367s
[INFO] 172.30.32.1:43166 - 56866 "PTR IN 242.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 132 0.003467755s
[INFO] 172.30.32.1:56214 - 11999 "PTR IN 102.10.168.192.in-addr.arpa. udp 45 false 512" NOERROR qr,aa,rd,ra 130 0.003698901s
```

So, there are no DNS errors in the log so WHY are there CONSTANT requests to 1.1.1.1 and 1.0.0.1 coming from the supervisor ?

![Screenshot from 2020-10-18 09-51-47](https://user-images.githubusercontent.com/59442445/96362796-c0255180-1127-11eb-84a1-6e8e426f2104.png)

**Comment by pvizeli** [comment]
Please read how Coredns and health checks works. The fallback bug fix 1.7.0 for unstable upstream DNS servers got fix until end of the year I think.

**Comment by tescophil** [comment]
Well, that doesn't help.., not much information there apart form the fact that the fallback plugin for CoreDNS is deprecated.

Are you saying this is a problem with Core DNS ?
